doc/source/install/cmake_options.rst

```diff
@@ -0,0 +1,89 @@
+.. -*- rst -*-
+
+CMake options
+=============
+
+This section describes important options of CMake.
+
+To get more details about installing Groonga from the source with CMake on a specific environment,
+find the document for the specific environment from :doc:`/install`.
+
+``-G GENERATOR``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Specify a generator.
+
+This is a mandatory option.
+
+You can check available generators by ``cmake --help``.
+
+Here is an example of specifying ``Visual Studio 12 2013 Win64`` as a generator::
+
+  > cmake . -G "Visual Studio 12 2013 Win64" 
+
+``-DCMAKE_INSTALL_PREFIX``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Specify a directory/folder to install Groonga.
+
+The default is depending on the system, e.g. `/usr/local` or `C:/Program Files (x86)/groonga`.
+
+Here is an example of specifying ``C:\Groonga`` as an install folder::
+
+ > cmake . -DCMAKE_INSTALL_PREFIX=C:\Groonga
```

Comment:
How about using `code-block:: console` or `code-block:: pwsh-session` for highlight instead of `::`?

```rst
.. code-block:: pwsh-session

   > cmake . -DCMAKE_INSTALL_PREFIX=C:\Groonga
```

BTW, why did you examples only for Windows? I think that most Windows users don't build Groonga by themself. Non-Windows users may build Groonga by themself.

Reply:
> BTW, why did you examples only for Windows?

Just because I have created this document based on `windows_cmake.rst`.

Reply:
I have fixed the examples for Linux except the example of  ``-G``.